osclib/stagingapi.py

```diff
@@ -1484,3 +1495,16 @@ def is_staging_bootstrapped(self, project):
             return meta.find(xpath) is not None
 
         return False
+
+    def delete_empty_adi_project(self, project):
+        if not self.is_adi_project(project):
+            return
+
+        if not self.is_staging_manager:
+            return
+
+        try:
+            delete_project(self.apiurl, project, force=True)
+        except HTTPError as e:
+            print(e)
```

Comment:
Why shouldn't it just `raise e`?

Reply:
I just moved this - but I still think it's a good idea to continue here. If the adi project is gone or not doesn't matter much, but if we exit the accept command, we don't cleanup the rest

Reply:
ok